### DISCUSSION TITLE
DataGrid is not displayed

### DISCUSSION THREAD

**Discussion by alvahtin** [AvaloniaUI/Avalonia#14292] [Q&A] [answered]
Visual Studio 2022, Avalonia 11.0.7

I builded library (Community Toolkit template), where userControl with DataGrid is created. I created new application (ReactiveUI template), setted link depending to my library project. But my DataGrid is'nt displayed. I installed package for the DataGrid in all projects. Other controls from my library is displayed, but not DataGrid.

Link to example my project, where is problem: https://github.com/alvahtin/FoundationLibrary

**Comment by timunie** [comment, marked as answer]
Make sure you have the StyleInclude done. Check DataGrid docs for reference

**Reply by alvahtin** [reply]
Thanks! I forgot about it!
